docker/Dockerfile.alpine

```diff
@@ -1,4 +1,4 @@
-FROM node:alpine
+FROM node:lts-buster-slim
```

Comment:
```suggestion
FROM node:alpine
```

Reply:
This change will not work. I tried to explain in #363 .